src/pages/Sample/index.tsx

```diff
@@ -25,10 +33,102 @@ const tabs = [
 const SamplePage: React.FC = () => {
   const accession = useURLAccession();
   const { data, loading, error } = useMGnifyData(`samples/${accession}`);
+  const [eezData, setEezData] = useState<EezMetadata>({
+    ...defaultEezMetadata,
+  });
+  const [fetchEezDataCalled, setFetchEezDataCalled] = React.useState(false);
   if (loading) return <Loading size="large" />;
   if (error) return <FetchError error={error} />;
   if (!data) return <Loading />;
   const { data: sampleData } = data as MGnifyResponseObj;
+
+  const fetchSovereignsAbsInfo = (mrgId: number) => {
+    const matchingEez = marineRegionsEezData.find((eez) => eez.MRGID === mrgId);
+
+    const sovereigns: SovereignsArray = [];
+
+    // const sovereigns: SovereignsArray = [
+    //   {
+    //     name: 'Brazil',
+    //     absStatus: 1,
+    //   },
+    //   {
+    //     name: 'France',
+    //     absStatus: 2,
+    //   },
+    //   {
+    //     name: 'United Kingdom',
+    //     absStatus: 5,
+    //   },
+    // ];
+
+    const maxPossibleNumberOfSovereigns = 3;
```

Comment:
Are there regions with many more sovereigns than this? Do we know which ones they will be and what the truncated sovereigns will be? 

Given this could be a contentious decision to truncate, if this is a common occurrence I think we might want to add something like `"and others"` or even just show a separate warning "This region has more than three sovereigns listed so ABS information is not being shown". If it is uncommon, do we need such a limit at all?

Reply:
@SandyRogers three is always the max number in the shapefile. 
There is no provision for more than 3 in the shapefile.